### DISCUSSION TITLE
`jax.block_until_ready` yields pyright warning

### DISCUSSION THREAD

**Discussion by alonfnt** [jax-ml/jax#12788] [Q&A] [answered]
Hi, 
I was wondering if there is a reason `block_until_ready` is not exported from module like the other functions (`block_until ready as block_until_ready`).

This prompts a pyright warning and I am not sure whether it is considered bad practice to use `jax.block_until_ready` directly.

Reference to the line:
https://github.com/google/jax/blob/0313b2241d7c81c69d06b031da322943952a9c95/jax/__init__.py#L66

Cheers

**Comment by hawkinsp** [comment, marked as answer]
I think that's just an oversight. Do you want to send a PR?

**Reply by alonfnt** [reply]
I can happily do that :)
